dev/com.ibm.ws.security.token.ltpa/resources/com/ibm/ws/security/token/ltpa/internal/resources/LTPAMessages.nlsprops

```diff
@@ -48,12 +48,28 @@ LTPA_CONFIG_ERROR=CWWKS4106E: LTPA configuration error. Unable to create or read
 LTPA_CONFIG_ERROR.explanation=The LTPA keys could not be read or created at the specified location.
 LTPA_CONFIG_ERROR.useraction=Ensure the location is accessible by the server process. Any FFDCs might indicate a fatal error in generating or loading the keys.
 
-LTPA_KEYS_TO_LOAD=CWWKS4107A: The LTPA keys file or configuration were modified. The LTPA keys will be reloaded from the file: {0}
-LTPA_KEYS_TO_LOAD.explanation=The LTPA keys need to be reloaded because there was a change to the LTPA keys file or the configuration.
-LTPA_KEYS_TO_LOAD.useraction=Ensure that all servers participating in Single-Sign-On use the same LTPA keys and password.
+LTPA_KEYS_TO_LOAD=CWWKS4107A: The LTPA keys files are either created, deleted, or modified: {0}, or the LTPA configuration was modified. All the LTPA keys will be reloaded.
+LTPA_KEYS_TO_LOAD=CWWKS4107A: Created, deleted, or modified LTPA key file: {0} or LTPA configuration were modified. All the LTPA keys files will be reloaded.
+LTPA_KEYS_TO_LOAD.explanation=The LTPA keys files need to be reloaded because there was a change to the LTPA keys files or the configuration.
+LTPA_KEYS_TO_LOAD.useraction=Ensure that all servers participating in Single-Sign-On share LTPA keys and password.
 
 LTPA_KEY_CREATE_ERROR=CWWKS4108E: The system cannot create the LTPA key.
 LTPA_KEY_CREATE_ERROR.explanation=The LTPA key cannot be created.
 LTPA_KEY_CREATE_ERROR.useraction=Any FFDCs might indicate a fatal error in generating the keys.
 
+LTPA_VALIDATION_KEYS_PASSED_NOT_USE_AFTER_DATE=CWWKS4109W: The notUseAfterDate attribute: {0} for the LTPA validation keys file: {1} is in the past. The LTPA validationKeys will not be used.
+LTPA_VALIDATION_KEYS_PASSED_NOT_USE_AFTER_DATE.explanation=The LTPA validation keys will be used only if the notUseAfterDate attribute is in the future.
+LTPA_VALIDATION_KEYS_PASSED_NOT_USE_AFTER_DATE.useraction=Update the notUseAfterDate attribute value to a future date if the LTPA keys have to be used. Otherwise, delete the LTPA keys file.
```

Comment:
```suggestion
LTPA_VALIDATION_KEYS_PASSED_NOT_USE_AFTER_DATE.useraction=Update the notUseAfterDate attribute value to a future date if the LTPA keys must be used. Otherwise, delete the LTPA keys file.
```